packages/textarea/src/styled.tsx

```diff
@@ -7,15 +9,63 @@ import {getValueWithUnit} from '@smashing/theme'
 type InputProps = TextareaProps &
   Required<Pick<TextareaProps, 'borderRadius' | 'appearance' | 'full'>>
 
-export const Textarea = styled(Text)<InputProps>`
+export const TextareaContainer = styled.div<InputProps>`
   border: none;
   border-radius: ${_ => getValueWithUnit(_.borderRadius)};
-  padding: 8px 10px;
+  padding: ${_ => (_.appearance === 'outline' ? '8px 10px' : 0)};
```

Comment:
Right now, when you click on that padded area, nothing will happen - textarea will not focus. Try to move padding to textarea, that should solve this issue but might create another one - check it

Reply:
Could you check this focus issue one more time? I've added `onClick` function that focuses textarea and it works well to me. Unfortunately, I can't move padding to textarea because it controls scroll position